integration/src/main/resources/configs/integration/ListWorkspaces.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "ListWorkspaces",
+  "description": "List workspaces",
+  "serverSpecificationFile": "workspace-dev.json",
```

Comment:
I think we can make all these `workspace-local.json` because this is overridden in the GHAs anyway.

Reply:
Good catch, I think that's a better default